build.py

```diff
@@ -0,0 +1,183 @@
+                    exported_file = export_directory / file_to_include.name
+                    exported_file.parent.mkdir(parents=True, exist_ok=True)
+                    try:
+                        # hardlinks take a fraction of time related to copy&paste
+                        # Unfortunately, some platforms don't implement or allow them for regular users (E.g. windows)
+                        file_to_include.link_to(exported_file)
+                        # Don't show the user a hard link is tried when the feature is not implemented
+                        printe('debug: hardlinked', file_to_include)
+                    except NotImplementedError:  # Not implemented in windows
+                        printe('debug: copy', file_to_include)
+                        shutil.copy(file_to_include, exported_file)
+                    except Exception:
+                        printe('debug: hardlink', file_to_include)  # Pretends the hardlink was tried but an error happened
+                        raise
+
+                    if zip:
+                        if file_to_include.suffix == '.js':
+                            printe('release: uglify js', file_to_include)
+                            zip_content = subprocess.check_output([
+                                'uglifyjs', '--compress', '--mangle', '--no-annotations', '-c', 'drop_console',
+                                '--', os.fspath(file_to_include.absolute())
+                            ])
+                        else:
+                            printe('release: include file', file_to_include)
+                            zip_content = file_to_include.read_bytes()
+
+                        zip.writestr(os.fspath(file_to_include), zip_content)
+
+                version_manifest = version / 'manifest.json'
+
+                printe('debug: manifest', version_manifest)
+
+                specific_manifest = json.loads(version_manifest.read_text())
+
+                specific_manifest['permissions'].extend(extension_manifest['permissions'])
+                extension_manifest.update(specific_manifest)
+
+                manifest_content = json.dumps(extension_manifest, indent=4)
+                (export_directory / 'manifest.json').write_text(manifest_content)
+
+                if zip:
+                    printe('debug: store manifest', version_manifest)
+                    zip.writestr('manifest.json', json.dumps(extension_manifest).encode('UTF-8'))
+
+    deploy()
+
+    if program_args.watch:
+        # All watchdog content here to "encapsulate" and only keep if wanted
+
+        from watchdog.observers import Observer
+        from watchdog.events import FileSystemEventHandler
+
+        class FileEventHandler(FileSystemEventHandler):
+
+            def on_modified(self, event):
+                if not event.is_directory and not os.fspath(EXPORT_PATH) in event.src_path:
+                    deploy()
+                    printe("updated")
```

Comment:
```suggestion
                    build()
                    printe("changes detected - rebuilt extension")
```
bit more verbose, better UX